common/wuanalysis/anawu.cpp

```diff
@@ -354,14 +352,6 @@ void WorkunitAnalyser::collateWorkunitStats(IConstWorkUnit * workunit, const WuS
     }
 }
 
-void WorkunitAnalyser::printWarnings()
```

Comment:
Better encapsulation to  keep this function and also add a createExceptions(wu) member, and avoids the refeference parameter on the constructor.